src/test/tests.ts

```diff
@@ -34,16 +34,17 @@ const mockOptions = {
 
 // -------------- Mocking Redis Client ------------------
 
-describe('RedisPubSub', function () {
+describe('RedisPubSub', () => {
 
-  it('should create default ioredis clients if none were provided', function (done) {
+  it('should create default ioredis clients if none were provided', done => {
     const pubSub = new RedisPubSub();
     expect(pubSub.getSubscriber()).to.be.an.instanceOf(IORedis);
     expect(pubSub.getPublisher()).to.be.an.instanceOf(IORedis);
+    pubSub.close();
```

Comment:
Can you please add an expect that will check that both the subscriber and publisher's `quit` methods were called?

Reply:
https://github.com/davidyaha/graphql-redis-subscriptions/pull/117/commits/bda0e18c076957ac15ce62b1777cd376e7c9d66f